package.json

```diff
@@ -33,9 +34,10 @@
     "access": "public"
   },
   "dependencies": {
-    "@optimizely/optimizely-sdk": "^5.0.0-beta2",
+    "@optimizely/optimizely-sdk": "5.0.0-beta2",
     "hoist-non-react-statics": "^3.3.0",
     "prop-types": "^15.6.2",
+    "tough-cookie": "4.1.3",
```

Comment:
Hmmm I'm now questioning this addition as an attempt to solve the Prisma CVE notification.  I don't think making this a direct dep is a solution.